.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -4,41 +4,70 @@ about: Create a report to help us improve TypeScript
 title: ''
 labels: ''
 assignees: ''
-
 ---
+# Bug Report
 
-<!-- 🚨 STOP 🚨 STOP 🚨 STOP 🚨
-
-Half of all issues filed here are duplicates, answered in the FAQ, or not appropriate for the bug tracker. Even if you think you've found a *bug*, please read the FAQ first, especially the Common "Bugs" That Aren't Bugs section!
+<!--
+  Please fill in each section completely. Thank you!
+-->
 
-Please help us by doing the following steps before logging an issue:
-  * Search: https://github.com/Microsoft/TypeScript/search?type=Issues
-  * Read the FAQ: https://github.com/Microsoft/TypeScript/wiki/FAQ
+### 🔎 Search Terms
 
-Please fill in the *entire* template below.
+<!--
+  What search terms did you use when trying to find an existing bug report?
+  List them here so people in the future can find this one more easily.
 -->
 
-<!--
-Please try to reproduce the issue with the latest published version. It may have already been fixed.
+### 🕗 Version & Regression Information
+
+<!-- When did you start seeing this bug occur?
+
+"Bugs" that have existed in TS for a long time are very likely to be FAQs; refer to
+  https://github.com/Microsoft/TypeScript/wiki/FAQ#common-bugs-that-arent-bugs
+
+If possible, please try testing the nightly version of TS to see if it's already been fixed.
 For npm: `typescript@next`
 This is also the 'Nightly' version in the playground: http://www.typescriptlang.org/play/?ts=Nightly
+
+Note: The TypeScript Playground can be used to try older verions of TypeScript.
+
+Please keep and fill in the line that best applies:
 -->
-**TypeScript Version:**  3.7.x-dev.201xxxxx
+ - This is a crash
+ - This changed between versions ______ and _______
+ - This is the behavior in every version I tried, and I reviewed the FAQ for entries about _________
+ - I was unable to test this on prior versions because _______
```

Comment:
These don't need to be indented

```suggestion
- This is a crash
- This changed between versions ______ and _______
- This is the behavior in every version I tried, and I reviewed the FAQ for entries about _________
- I was unable to test this on prior versions because _______
```